### PR TITLE
Made profile page and song pop up not hard coded.

### PR DESCRIPTION
**Pull request by izakbunda**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,94 +1,91 @@
...
   return (
-    //<Home />
     <NavigationContainer>
-      <Stack.Navigator initialRouteName="Landing">
-        <Stack.Screen 
-          name="Landing" 
-          component={Landing} 
-          options={{headerShown: false}}
-          />
+      <Stack.Navigator initialRouteName="Home">
```
Comment (jchangz01) [inline App.js:21]: Can you please change the initialRouteName back to "Landing"

File: App.js [hunk 2]
```diff
@@ -1,94 +1,91 @@
...
           name="Signup"
           component={Signup}
-          options={{headerShown: false}}
+          options={{ headerShown: false }}
         />
         <Stack.Screen
           name="Home"
           component={Home}
-          options={{headerShown: false}}
+          options={{ headerShown: false }}
```
Comment (jchangz01) [inline App.js:35]: You can go ahead and remove the "Home" Stack.screen (Lines 32-36)

File: App.js [hunk 3]
```diff
@@ -1,94 +1,91 @@
...
         />
```
Comment (jchangz01) [inline App.js:36]: You can go ahead and remove the "Home" Stack.screen (Lines 32-36)

**Review by jchangz01 — CHANGES_REQUESTED** [review]
Make the changes in App.js and the rest should be fine

**Comment by izakbunda** [comment]
Changes addressed.

**Comment by jchangz01** [comment]
Ok, it seems that Jake already created a component called Playlist.js. As a result, there is a merge conflict. To fix this, I think you should change your Playlist.js to ProfilePlaylist.js. And then utilize Jake's PlaylistImage.js component (you will need to rebase dev to see these changes) in your ProfilePlaylist.js line 16. Let me know if you have any questions.

**Pull request merged**